testing/src/main/java/org/motechproject/nms/testing/service/impl/TestingServiceImpl.java

```diff
@@ -174,100 +212,481 @@ public TestingServiceImpl() {
     }
 
 
+    private void truncateTable(final String table) {
+        SqlQueryExecution sqe = new SqlQueryExecution() {
+
+            @Override
+            public String getSqlQuery() {
+                return String.format("DELETE FROM %s WHERE 1=1", table);
```

Comment:
It's a trick to avoid getting an "you can't do a global delete on a table without a where clause" error.